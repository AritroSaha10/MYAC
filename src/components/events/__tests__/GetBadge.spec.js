import renderer from "react-test-renderer"

import getBadge from "../GetBadge"

describe("Badge function", () => {
    test("Returns the correct badge", () => {
        const tree = renderer
            .create(getBadge("Concluded"))
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})