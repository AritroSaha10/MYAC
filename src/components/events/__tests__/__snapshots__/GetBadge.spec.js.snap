// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge function Returns the correct badge 1`] = `
<span
  className="chakra-badge css-ifsvay"
>
  Concluded
</span>
`;
